src/routes/SwapLightning.tsx

```diff
@@ -146,7 +146,7 @@ export function SwapLightning() {
             setStage("preview");
         } catch (e) {
             console.error(e);
-            setFeeEstimateWarning(i18n.t("swap_lightning.too_small"));
+            setFeeEstimateWarning(eify(e));
```

Comment:
can keep the existing functionality:

```suggestion
            const err = eify(e);
            if (err.message === "Satoshi amount is invalid") {
                setFeeEstimateWarning(
                    new Error(i18n.t("swap_lightning.too_small"))
                );
            } else {
                setFeeEstimateWarning(err);
            }
```